lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
Ok, fine. Though I don't understand why `background` is different from `format`. Do you think will be re-used for sure by other template implementations?

Perhaps I'm missing something and this is a dumb queston, but say I want to write a template for this layout (handling the `t.maybe(t.Date)` type):

```
+------------+---+---+---+---+
| 1973-11-30 | X | ^ | + | - |
+------------+---+---+---+---+
```

where

- `X` is a button that removes the current date
- `^` is a button that opens a picker to change the current date
- `+` is a button that adds a day to the current date
- `-` is a button that subtracts a day to the current date

Will the `background` option be re-used? For which of those 4 buttons? Seems weird.